src/main/java/mate.academy/Main.java

```diff
@@ -10,7 +10,9 @@ public class Main {
     public static void main(String[] args) {
         // Please test your Injector here. Feel free to push this class as a part of your solution
         Injector injector = Injector.getInjector();
-        ProductService productService = null;
+        Object instance = injector.getInstance(ProductService.class);
+        System.out.println(instance.getClass().getName());
+        ProductService productService = (ProductService) injector.getInstance(ProductService.class);
```

Comment:
You have already had `instance` as an implementation of `ProductService.class` Just cast it here, instead of getting one more time

Reply:
refactored